ova/README.md

```diff
@@ -0,0 +1,64 @@
+Wazuh
+=====
+
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://wazuh.com/community/join-us-on-slack/)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
+
+In this repository, you can find the necessary tools to build your own OVA file with all Wazuh componets installed.
+
+## Tools needed to build the OVA:
+
+To build an OVA you need to install the following tools:
+- `Virtual Box`: [instalation guide](https://www.virtualbox.org/manual/UserManual.html#installation)
+- `Vagrant`: [instalation guide](https://www.vagrantup.com/docs/installation/)
+- `Git`:  [installation guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
+- `Python`
+
+
+## Building OVA file:
+
+To generate the ova file, you need to download this repository and use the `generate_ova.sh` script. This script will create a vagrant box and provision it to be ready to use wazuh, as specified on provision.sh, and then export it to an Ova file.
+
+```shellsesion
+
+# ./generate_ova.sh -h
+
+  OPTIONS:
+      -b, --build            [Required] Build the OVA and OVF.
+      -v, --version          [Required] Version of wazuh to install on VM.
+      -e, --elastic-version  [Required] Elastic version to download inside VM.
+      -r, --repository       [Required] Status of the packages [stable/unstable]
+      -c, --clean            [Optional] Clean the local machine.
+      -h, --help             [  Util  ] Show this help.
+
+```
+
+  
+ To build an OVA with versiohn 3.9.0 using elastic 6.2.2 and the stable repositories you can use:
```

Comment:
Please fix 6.2.2 version

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/162/commits/99bd73a341721442e0fc065982bd83743368562d